lib/autocomplete-provider.js

```diff
@@ -107,7 +107,14 @@ export default function() {
         return null;
       }
 
-      if (prefix.trim().length < 3) {
+      var minimumWordLength = atom.config.get(
```

Comment:
I'd prefer `let` over `var` here. We tend to use `let` and `const` in the codebase, so it would be nice to keep it consistent.